docs/src/main/asciidoc/transaction-guide.adoc

```diff
@@ -166,6 +166,11 @@ public class SantaClausService {
 You cannot use `UserTransaction` in a method having a transaction started by a `@Transactional` call.
 ====
 
+== Configuring the transaction timeout
+You can configure the default transaction timeout, the timeout that will applies to all transactions managed by the transaction manager, via the property `quarkus.narayana.default-transaction-timeout` (in seconds). The default value is unspecified in Quarkus, this means that the default of the transaction manager implementation is used.
```

Comment:
```suggestion
You can configure the default transaction timeout, the timeout that will apply to all transactions managed by the transaction manager, via the property `quarkus.narayana.default-transaction-timeout` (in seconds). The default value is unspecified in Quarkus, this means that the default of the transaction manager implementation is used.
```

Reply:
It will get reorganized but, just for future work: add a new line at the end of each sentence. Asciidoctor will make it a single paragraph anyway and it makes diffs more readable.

Reply:
rephrased